docs/site/components/subpage/subpage-directive.js

```diff
@@ -11,12 +11,16 @@ angular
         $scope.links = $parse(attrs.links)();
         $scope.headerTemplateUrl = attrs.headerTemplateurl;
 
+        document.querySelector('.nav-current').onclick = function(){
+          document.querySelector('.main-nav').classList.toggle('open');
+        };
+
```

Comment:
You can do this without JS at all actually. Check out https://github.com/GoogleCloudPlatform/gcloud-node/blob/master/docs/site/components/docs/docs.html#L19 for an example of using ng-click to set a Boolean and google ng-class to see how to set a class based on a Boolean.